examples/argo/Makefile

```diff
@@ -6,6 +6,14 @@ ARGO_SECRET = $(shell kubectl -n ${ARGO_NAMESPACE} get secret argocd-initial-adm
 
 .PHONY: install
 install:
+	@echo "-----------------------------------"
+	@echo "Create Namespace and install Keptn-lifecycle-toolkit"
+	@echo "-----------------------------------"
+	kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.10.0/cert-manager.yaml
+	kubectl wait --for=condition=available deployment/cert-manager -n cert-manager --timeout=300s
+	kubectl apply -f https://github.com/keptn/lifecycle-toolkit/releases/download/v0.4.0/manifest.yaml
```

Comment:
you could add the release please annotation here to update the version automatically